mpv.el

```diff
@@ -373,5 +373,37 @@ the echo area."
       (if (called-interactively-p 'interactive)
 	  (message "mpv %s" version)))))
 
+;;;###autoload
+(defun mpv-playlist-play-index (pos)
+  "Start (or restart) playback of the given playlist index.
+
+This function provides a backward compatible wrapper of the mpv
+command playlist-play-index which has been introduced in mpv
+version 0.33.
+
+The value of POS can be either
+- a 0-based positive number: the entry of index POS will be played (or replayed
+if it corresponds to the current playlist entry);
+- \"current\": the current playlist entry will be played again;
+- \"none\": playback is stopped.
+
+For more information, see the documentation:
+https://mpv.io/manual/master/#command-interface-playlist-play-index"
+  (if (version<= "0.33" (mpv-version))
+      (mpv-run-command "playlist-play-index" pos)
+    (let ((cur (mpv-get-property "playlist-pos")))
+      (pcase pos
+	("current" (mpv-seek 0))
+	((and (pred numberp) (pred (= cur))) (mpv-seek 0))
+	("none" (mpv-set-property "pause" "no"))
+	((and (pred numberp) (pred (<= 0)))
+	 (cl-loop repeat (abs (- pos cur))
```

Comment:
According to the linked documentation (https://mpv.io/manual/master/#command-interface-playlist-pos), `cur` might be `-1` if no entry is "current". This is not handled at the moment, right? Also:
> Before mpv 0.33.0, instead of returning -1, this property was unavailable if no playlist entry was current.

My version of mpv is too new to test this, but my assumption is that `mpv-get-property` would fail with `‘get_property’ failed: property not found` in that case?

Reply:
Also, the mpv documentation indicates that `playlist-pos` is writable? So it _might_ be possible to replace the `cl-loop` with `(mpv-set-property "playlist-pos" pos)` in the `(!= pred cur)` case. Can you give it a try on your version of mpv?